ClientApp/src/Pages/Conservation/Conservation.scss

```diff
@@ -0,0 +1,40 @@
+@import "~bootstrap/scss/bootstrap";
+
+body {
+  font-family: "Segoe UI";
+}
+$margin: 30px 30px;
+$margin-top: 30px;
+$margin-bottom: 30px;
+
+.conservationTitle {
```

Comment:
tiny thing, but to be consistent with everything else I think this should be `conservation-title`

Reply:
updated